app/controllers/trajectories_controller.rb

```diff
@@ -0,0 +1,34 @@
+class TrajectoriesController < ApplicationController
+    before_action :set_trajectory, only [show update destroy]
+    
+    #met Get
+    def
+        @trajectory = Trajectory.indCluides(:schedules). find(parms [:id])
+        render json: @trajectory.to_json(include: :schedules)
+    end
+
+    #post /trajectories 
+
+    def create
+        @trajectory = Trajectory.new(trajectoy_params)
+        if @trajectory.save
+            render json: @trajectory, status: :created
+        else 
+            render json: @trajectory.errors status: :unprocessable_entity
+        end
+    end
+    # Use callbacks to share common setup or constraints between actions.
+
+    private 
+
+    def set_trajectory
+        @trayectory = Trajectory.find(params[:id])
```

Comment:
What would it happend when the record is not found?

Reply:
I would make a mistake sorry